scylladb/scylla-grafana-monitoring
# Lint documentation configuration

Related https://github.com/scylladb/sphinx-scylladb-theme/issues/133

## Context

`scylla-monitoring` was one of the first projects to receive the Scylla Sphinx Theme 1.0 update.

After adding the docs to other projects, we have improved the GitHub Actions workflows and organized the ``conf.py`` similarly for all projects. This PR applies these changes to the monitoring project.

Additionally, this PR rollback the custom index page added in https://github.com/scylladb/scylla-monitoring/pull/1558.
The reason for changing the index page (again) is that we noticed the landing page does render nicely when the site's sidebar is enabled:

![image](https://user-images.githubusercontent.com/9107969/138449287-b7fd141c-c997-4e37-a05b-f5378a3050d4.png)

Before adding the custom landing page back, we need to implement first https://github.com/scylladb/sphinx-scylladb-theme/pull/230 in the next theme version.

 ## How to test this PR
 
 1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser. You will see the docs and the terminal should not raise any warnings.
